core/objects.go

```diff
@@ -56,6 +59,18 @@ const (
 	IdentifierDNS = IdentifierType("dns")
 )
 
+// The types of ACME resources
+const (
+	ResourceNewReg        = AcmeResource("new-reg")
+	ResourceNewAuthz      = AcmeResource("new-authz")
+	ResourceNewCert       = AcmeResource("new-cert")
+	ResourceRevokeCert    = AcmeResource("revoke-cert")
+	ResourceRegistration  = AcmeResource("reg")
+	ResourceAuthorization = AcmeResource("authz")
```

Comment:
Good catch. Since the spec does not call for returned objects to contain a resource field, these are not needed. @bifurcation can you remove?